buildspec.json

```diff
@@ -48,12 +48,11 @@
             "bundleId": "fr.palakis.obs-ndi"
         }
     },
-    "name": "obs-ndi",
-    "displayName": "OBS-NDI",
-    "version": "4.14.0",
-    "website": "https://obsproject.com/forum/resources/obs-ndi-newtek-ndi%E2%84%A2-integration-into-obs-studio.528/",
-    "author": "Stéphane Lepin",
-    "email": "stephane.lepin@gmail.com",
+    "name": "DistroAV",
+    "version": "6.0.0",
+    "website": "https://distroav.org",
+    "author": "DistroAV",
+    "email": "distroav@distroav.org",
```

Comment:
Wasn't it agreed to have contact@distroav.org ?

Reply:
Briefly discussed on Discord, but not formally agreed.

There are two different contexts for the two different uses of the email address.
1. buildspec.json : main standard "email address of the author of this plugin" type of "play store" contact info
2. copyright/legal headers

I considered all of the following (and already setup free email forwards for):
* code@distroav.org
* contact@distroav.org
* copyright@distroav.org
* dev@distroav.org
* distroav@distroav.org
* foss@distroav.org
* info@distroav.org
* legal@distroav.org
* license@distroav.org
* ...

I had already committed and pushed `distroav@distroav.org` when you mentioned in Discord to use `contact@distroav.org`

I personally liked the more warm/personable/approachable "my name is DistroAV and you can email me at `distroav@distroav.org`" over the more cold/formal/business of `contact@distroav.org`.

I can change it to `contact@distroav.org`, but I'd like a 3rd opinion on this fairly anchoring decision.  
@RayneYoruka , what do you think?